src/control/Control.Zoom.js

```diff
@@ -68,14 +69,18 @@ export var Zoom = Control.extend({
 	},
 
 	_zoomIn: function (e) {
+		var zoomDelta = this._map.options.zoomDelta;
+		var delta = any3d ? zoomDelta : Math.max(1, zoomDelta);

```

Comment:
Once again, this will misbehave in an edge case. Think about what happens when `zoomDelta = 1.5`.

Reply:
@IvanSanchez 
The main problem was with zoomDelta less then 0.5 with disabled 3D because if you take a look at [_limitZoom](https://github.com/Leaflet/Leaflet/blob/master/src/map/Map.js#L1576) method it rounds new zoom level. It means if zoomDelta equals 0.25 buttons for zoom in/out and double click won't work because with zoom level 1 will be increased to zoom level 1.25 and rounded in _limitZoom method to 1. It means nothing will be changed. But if zoomDelta will be more then 0.5 it will work but rounded to closes integer. 

in your case if zoomDelta = 1.5 with zoom level 1 when user  clicks on zoom in button in controls zoom will be increased to 2.5, but _limitZoom method will round it to 3.

Reply:
Makes sense to me